espn_ffb/app.py

```diff
@@ -47,9 +48,37 @@ def percentage_format(value):
     return "{:.4f}".format(value)
 
 
+def get_template_path_vars_for_nav(relative_path, pattern, splitter=None):
+    matching_paths = glob.glob(relative_path + pattern)

```

Comment:
There's probably a _better_ way to do this where the static contents get stored in the db as opposed to individual html files. But this implementation works without requiring existing leagues/maintainers to migrate their templates.

Reply:
Have also played around with the idea of storing static contents in the db. Maybe a one-off script for existing users to migrate templates to db.

This commit broke **Awards** and **Playoffs** nav for me.

Reply:
Fixed in https://github.com/raphattack/espn-ffb/pull/13/commits/dbaa3b42af8aeda34588c333c3cae9968e91f1a6